blocks/blockoracle.go

```diff
@@ -111,7 +111,7 @@ func (bo *Oracle) calcEligibilityProofs(epochNumber types.EpochID) (map[types.La
 
 	bo.log.With().Info("Got beacon",
 		log.Uint64("epoch_id", uint64(epochNumber)),
-		log.Err(err))
+		log.String("epoch_beacon", fmt.Sprint(epochBeacon)))
```

Comment:
is this a binary value? it will be better to use log.Binary, it will encode it in base64 and won't copy memory until the moment when it needs to be printed. unfortunately it is not that easy to extend zap to encode binary to hexary instead of base64

Reply:
actually i copied it from a Debug log below. i'm going to change it to
```
types.BytesToHash(epochBeacon).ShortString()
```
to be consistent with the rest of the code. thanks for the reminder 💯

Reply:
that is still not perfect btw :) this will copy and encode this field even if it is not printed later